src/nvim/ex_getln.c

```diff
@@ -1863,7 +1863,10 @@ char *getcmdline_prompt(const char firstc, const char *const prompt,
   ccline.input_fn = (firstc == '@');
   ccline.highlight_callback = highlight_callback;
 
+  int msg_silent_saved = msg_silent;
+  msg_silent = 0;
   char *const ret = (char *)getcmdline(firstc, 1L, 0);
+  msg_silent = msg_silent_saved;
 
   restore_cmdline(&save_ccline);
```

Comment:
in the vim patch, this comes before the restore of msg_silent. Doesn't matter, but  good to keep it consistent unless there's a reason not to.